p12/src/main.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    println!("Hello, world!");
+    let a = 42u64;
+    let _ = a as u64;
```

Comment:
This still triggers the Clippy warning. You can fix it by simply removing these two lines.